github/resource_github_repository_tag_protection_test.go

```diff
@@ -0,0 +1,151 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccGithubRepositoryTagProtection(t *testing.T) {
```

Comment:
Integration tests are unfortunately not run on CI for now but they pass for me locally!